portfolio/src/main/webapp/script.js

```diff
@@ -47,11 +47,44 @@ document.addEventListener('DOMContentLoaded', function() {
       navbar.classList.remove('sticky');
     }
   }
+})
+/**
+ * Fetches comments from the server and adds them to the DOM.
+ */
+function loadComments() {
+  fetch('/comment').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
```

Comment:
have you run clang-format on all your files before your last commit?

Reply:
I ran clang-format on my last commit.